src/ai-bundle/config/options.php

```diff
@@ -375,6 +375,19 @@
                             ->end()
                         ->end()
                     ->end()
+                    ->arrayNode('supabase')
+                        ->useAttributeAsKey('name')
+                        ->arrayPrototype()
+                            ->children()
+                                ->scalarNode('url')->isRequired()->cannotBeEmpty()->end()
```

Comment:
Following #604, please make the http client configurable, you can barely copy paste from #604, thanks

Reply:
I do not see it done for any other store. Am I missing something here?